commons/src/methods/isFileAnAttachment.ts

```diff
@@ -1,15 +1,7 @@
 import type { Message, File } from "@commons/types/Nylas";
-import {
-  DisallowedContentTypes,
-  InlineImageTypes,
-} from "@commons/constants/attachment-content-types";
+import { DisallowedContentTypes } from "@commons/constants/attachment-content-types";
 
 export const isFileAnAttachment = (message: Message, file: File): boolean =>
   file.content_disposition === "attachment" &&
-  file.filename &&
-  !(
-    file.content_id &&
-    message.cids?.includes(file.content_id) &&
-    InlineImageTypes.includes(file.content_type)
-  ) &&
```

Comment:
Removed the filter for Inline image types because image can also be an attachment.

And I also removed the content_id filter because emails thread sent with outlook doesn't have content id.

Reply:
Looks good to me.